doc/source/notebooks/advanced/ordinal_regression.pct.py

```diff
@@ -109,7 +109,7 @@ def pred_log_density(m):
         Ytest = np.full_like(Xtest, y)
         # Predict the log density
         densities.append(m.predict_log_density((Xtest, Ytest)))
```

Comment:
I can't comment on the line but there was another `predict_density` that should be `predict_log_density` in this file on line 133

Reply:
done